flow/layers/shader_mask_layer_unittests.cc

```diff
@@ -373,17 +373,17 @@ TEST_F(ShaderMaskLayerTest, OpacityInheritance) {
     {
       expected_builder.translate(offset.fX, offset.fY);
       /* ShaderMaskLayer::Paint() */ {
-        expected_builder.setColor(opacity_alpha << 24);
-        expected_builder.saveLayer(&child_path.getBounds(), true);
+        DlPaint sl_paint;
+        sl_paint.setColor(opacity_alpha << 24);
```

Comment:
These could be chained into 1 line if you want (up to you), as in `= DlPaint().setColor(...);`

Reply:
Actually, you can use `setAlpha(opacity_alpha);` This method appears on DlPaint, but it wasn't on Builder...